LAKMobile/src/screens/SignupScreen.tsx

```diff
@@ -0,0 +1,115 @@
+import { StyleSheet, View, TextInput } from 'react-native';
+import { PrimaryButton, LabelWrapper, AppText } from '../components';
+import { COLORS } from '../../constants';
+
+export default function SignupScreen({}) {
+  return (
+    <View style={styles.container}>
+      <LabelWrapper label='Name (First, Last)'>
+        <TextInput
+          style={styles.input}
+          keyboardType="default"
+        />
+      </LabelWrapper>
+      
+      <LabelWrapper label='Mobile Number'>
+        <TextInput
+          style={bigInputStyle}
+          keyboardType="default"
+        />
+      </LabelWrapper>
+
+      <LabelWrapper label='Location'>
+        <TextInput
+          style={bigInputStyle}
+          keyboardType="default"
+        />
+      </LabelWrapper>
+
+      <LabelWrapper label='4 digit pin password'>
+        <TextInput
+          style={smallInputStyle}
+          keyboardType="numeric"
+        />
+      </LabelWrapper>
+
+      <LabelWrapper label='Confirm password'>
+        <TextInput
+          style={smallInputStyle}
+          keyboardType="numeric"
+        />
+      </LabelWrapper>
+
+      <PrimaryButton
+        type='primary'
+        title='Create Account'
+        onPress={() => console.log('Create Account pressed')}
+        style={styles.submitButton}
+        />
+
+      <View style={styles.loginLinkContainer}>
+        <AppText>Already have an account?</AppText>
+        <PrimaryButton 
+          type='link' 
+          title='Log in here' 
+          onPress={() => console.log('Log in here pressed')}
+          style={styles.loginLink}
+          />
+      </View>
+    </View>
+  );
+}
+
+// Page Styling
+const styles = StyleSheet.create({
+  container: {
+    width: '100%',
+    padding: 32,
+    flex: 1,
+    backgroundColor: COLORS.white,
+    alignItems: 'flex-start',
+    justifyContent: 'center',
+  },
+  
+  // Shared across all inputs
+  input: {
```

Comment:
Other components will use these styles for the input so moving the input styling to an outside file might be better.

Reply:
Plan is to move all inputs into a single component. Since this PR introduces components that multiple other open PRs depend on maybe we could create a separate issue for the input component and merge this PR. Lemme know your thoughts?